src/PrestaShopBundle/Controller/Admin/AuthorizationController.php

```diff
@@ -107,18 +108,21 @@ public function forgotPasswordAction(Request $request)
         $showForgotPasswordForm = true;
 
         if ($forgotPasswordForm->isSubmitted() && $forgotPasswordForm->isValid()) {
+            $successMessage = $this->trans(
+                'Check your mailbox and click on the link to reset your password.',
+                'Admin.Login.Notification'
+            );
+
             try {
                 $this->getCommandBus()->handle(
                     new SendResetPasswordEmailCommand($forgotPasswordForm->getData()['email'])
                 );
 
-                $this->addFlash(
-                    'success',
-                    $this->trans(
-                        'Check your mailbox and click on the link to reset your password.',
-                        'Admin.Login.Notification'
-                    )
-                );
+                $this->addFlash('success', $successMessage);
```

Comment:
Can we store `success`, `error`, `info`, etc. To prevent typo error like `sucess`?

Reply:
in my opinion it would make code less clean :confused: 
if we store the constants in the `FrameworkBundleAdminController` or somewhere else, this line:
`$this->addFlash('success', $successMessage);`
would become this:
`$this->addFlash(FrameworkBundleAdminController::SUCCESS, $successMessage);`
which is very long and quite redundant (almost double the length) :confused:

let me know your opinion @PierreRambaud

Reply:
We don't care if it's long or not (under 100 chars ^^') and `'sucess'` is also redundant.
I just care about `sucess`, `eror`, `warming` :/
Plus it's faster because of const pointer. 
But it's just a suggestion, wdyt @matks ?

Reply:
I agree with @PierreRambaud . Code length is a small price to pay in comparison of the benefits of not having a bug in production and not having to diagnose it and not having to fix it 😄 

This is the same reason that Symfony used to decide whether to create consts `Response::HTTP_CREATED` instead of the classic 401 or `Response:: HTTP_OK` instead of the classic 200
https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Response.php

Reply:
@matks @PierreRambaud what if we had methods for adding specific messages, like  `FrameworkBundleAdminController::addSuccess($message)`, `FrameworkBundleAdminController::addError($message)`?

Reply:
We maybe should be able to do both:
```php
$this->addSuccess($message);
$this->flashBag(self::FLASH_TYPE_SUCCESS, $message);
```